### PR TITLE
Removed apostrophe from MicroMaster's

### LINKED ISSUE #557: Remove the Apostrophe from MicroMaster's
The News Office has reconsidered and MicroMaster's does not need to have an apostrophe everywhere. 

Presumably, all that's required is a revert of #478 and #236, but there may be a few instances in the text beyond those two PRs. Also, if an apostrphe make sense in the context, we can leave it in.

### PR DESCRIPTION
**Pull request by noisecapella**
#### What are the relevant tickets?
Fixes #557 

#### What's this PR do?
Removes apostrophe from MicroMaster's to become MicroMasters. Some apostrophes are left if they make sense in context

#### How should this be manually tested?
Grep through the code base to make sure all existing uses are updated, if relevant to the context

### CODE REVIEW

File: static/js/components/Footer.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Footer extends React.Component {
          <div className="container container-footer">
             <div className="row">
                <div className="col-md-8">
-                  <img src="/static/images/mit-logo-ltgray-white@72x38.svg" alt="MicroMaster’s" />
+                  <img src="/static/images/mit-logo-ltgray-white@72x38.svg" alt="MicroMasters" />
```
Comment (pdpinch) [inline static/js/components/Footer.js]: Seems like this alt tag should actually be "MIT"

File: cms/templates/cms/footer.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
      <div class="container">
         <div class="row">
            <div class="col-md-8">
-              <img src="/static/images/mit-logo-ltgray-white@72x38.svg" alt="MicroMaster’s">
+              <img src="/static/images/mit-logo-ltgray-white@72x38.svg" alt="MicroMasters">
```
Comment (pdpinch) [inline cms/templates/cms/footer.html]: Seems like this alt tag should actually be "MIT"

File: static/js/containers/TermsOfServicePage.js [hunk 3]
```diff
@@ -59,8 +59,8 @@ class TermsOfServicePage extends React.Component {
                 <ol>
                   <li>
                     Your use of this Site is entirely voluntary.  This Site is designed to assist and facilitate your
-                    successful attainment of the MITx MicroMaster’s credential, but use of this Site is not required
-                    in order to participate in the MITx MicroMaster’s program. These terms and conditions govern
+                    successful attainment of the MITx MicroMaster's credential, but use of this Site is not required
```
Comment (pdpinch) [inline static/js/containers/TermsOfServicePage.js]: remove single quote. I think MicroMasters is used more as a adjective here than a possessive

**Comment by pdpinch** [comment]
👍 after a rebase

**Comment by pdpinch** [comment]
a couple of small things, but otherwise looks good

**Comment by pdpinch** [comment]
whoops, didn't realize this was still in progress

**Comment by noisecapella** [comment]
It's just in progress so we could discuss the questions in the linked issue. Did you have any feedback on those questions?

File: cms/templates/cms/home_page.html [hunk 4]
```diff
@@ -138,20 +138,19 @@
...
           <p>
-          MicroMaster’s aren't just about courseware. Part of the the MIT experience is interacting with other exceptional students like yourself. Fill out your profile on this site and build professional relationships with like-minded learners.
+          MicroMasters aren't just about courseware. Part of the the MIT experience is interacting with other exceptional students like yourself. Fill out your profile on this site and build professional relationships with like-minded learners.
           </p>
-          <h3>MicroMaster’s courses are delivered through edX.org</h3>
+          <h3>MicroMasters courses are delivered through edX.org</h3>
           <p>
-          Use your edX.org account to enroll in a MicroMaster’s program and share your grades and progress with MIT. When you complete all the requirements and earn a MicroMaster’s credential, you can use it to apply for a Masters degree at MIT.
+          Use your edX.org account to enroll in a MicroMasters program and share your grades and progress with MIT. When you complete all the requirements and earn a MicroMaster’s credential, you can use it to apply for a Masters degree at MIT.
```
Comment (aliceriot) [inline cms/templates/cms/home_page.html]: is the second `MicroMaster's` here retaining an apostrophe intentionally?

File: static/js/components/PersonalTab.js [hunk 5]
```diff
@@ -20,7 +20,7 @@ class PersonalTab extends React.Component {
     return <div>
       <Grid className="profile-splash">
         <Cell col={12}>
-          Please tell us more about yourself so you can participate in the MicroMaster’s
+          Please tell us more about yourself so you can participate in the MicroMasters
           community and qualify for your MicroMaster’s credential.
```
Comment (aliceriot) [inline static/js/components/PersonalTab.js]: should this be `MicroMasters credential`?

**Comment by aliceriot** [comment]
few questions, looked like there's a merge conflict as well.

File: static/js/components/PersonalTab.js (hunk 5 above)
Comment (noisecapella) [inline static/js/components/PersonalTab.js]: Yes, I talked with @pdpinch earlier and MicroMaster's degree is the only case where there should be an apostrophe

Comment (aliceriot) [inline static/js/components/PersonalTab.js]: ok cool

**Comment by noisecapella** [comment]
Ok, all comments should be addressed

**Comment by aliceriot** [comment]
:+1: :shipit: :confetti_ball: :fireworks: :ok_hand:

**Pull request merged**